media-libs/imlib2/imlib2-1.5.1-r1.ebuild

```diff
@@ -67,9 +67,11 @@ multilib_src_install() {
 
 	if use doc; then
```

Comment:
It was failing because it belongs in multilib_src_install**_all**.

Reply:
So, would something like this be correct then:
```
multilib_src_install() {
    V=1 emake install DESTDIR="${D}"
    find "${D}" -name '*.la' -delete || die
}

multilib_src_install_all() {
    default

    if use doc; then
        local HTML_DOCS=( "${S}"/doc/. )
        rm "${S}"/doc/Makefile.{am,in} || die
    fi
    einstalldocs
}
```

(Yes, I tested that it works)

Reply:
Without the `default` in _all, it's calling make+einstalldocs.

Reply:
Done, seems to work.
```
multilib_src_install() {
    V=1 emake install DESTDIR="${D}"
    find "${D}" -name '*.la' -delete || die
}

multilib_src_install_all() {
    if use doc; then
        local HTML_DOCS=( "${S}"/doc/. )
        rm "${S}"/doc/Makefile.{am,in} || die
    fi
    einstalldocs
}
```

Will push an update and rebase in a minute.